src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/Details/components/DetailInput.tsx

```diff
@@ -9,6 +9,7 @@ interface DetailInputProps {
   field?: RegularFieldType
   formikProps: FormikProps<EditQuoteInput>
   nameRoot:
+    | ''
```

Comment:
Went back and forth between having nameRoot optional, name optional but I thought this was the best solution since this is more in line with how its used. You should pass a root and you should pass a name. I think having this as optional makes it unclear how to use this component.

Reply:
I see your point :) But I think it's odd with passing an empty string as `nameRoot`. I think I would prefer having it as optional but not a big deal to me if we implement it like this

Reply:
I thought about again.  The point, that enforcing nameRoot helps in understanding the component, is maybe not that valid.... At least not worth it. Changing it to optional! 😆